content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -56,14 +56,15 @@ const NewTabPage = React.createClass({
           title="Welcome to new tab"
           body="Firefox will use this space to show your most relevant bookmarks, articles, videos, and pages you've recently visited, so you can get back to them easily."
           label="Identifying your Highlights" />
```

Comment:
When testing this, I noticed there was a delay before the loader appeared. Is this 2 second delay desired? Probably want to have a followup bug to figure that out. It was introduced here:

https://github.com/mozilla/activity-stream/commit/54ea1f816c5a84ca13466df53077aee9a96e01ad